xapian-core/cluster/cluster.cc

```diff
@@ -369,8 +366,8 @@ Cluster::clear() {
     cluster_docs.clear();
 }
 
-Centroid
-Cluster::get_centroid() const {
+Centroid&
+Cluster::get_centroid() {
```

Comment:
The problem I had here was that a const function cannot return a reference. Am I doing something wrong here?